src/ucp/api/ucp.h

```diff
@@ -4518,17 +4520,29 @@ typedef struct ucp_ep_attr {
      * this name.
      */
     char     name[UCP_ENTITY_NAME_MAX];
+
+    /**
+     * The local sockaddr that this ep is using.
+     */
+    struct sockaddr_storage local_sockaddr;
+
+    /**
+     * The remote sockaddr this ep is connected to.
+     */
+    struct sockaddr_storage remote_sockaddr;
 } ucp_ep_attr_t;
 
 
 /**
- * @ingroup UCP_ENDPOINT
- * @brief Get attributes specific to a particular endpoint.
+ * @ingroup UCP_WORKER
+ * @brief Get attributes of a given endpoint.
  *
  * This routine fetches information about the endpoint.
+ * It should be called only if the endpoint was created with the
+ * UCP_EP_PARAM_FIELD_SOCK_ADDR or UCP_EP_PARAM_FIELD_CONN_REQUEST flags.
```

Comment:
pls remove since some of endpoint attributes are not related to sockaddr

Reply:
@yosefe  Done